docs/UserGuide.adoc

```diff
@@ -1005,46 +1235,52 @@ Spelling errors can be mis-interpreted and the wrong command might be invoked!
 
 == Command Summary
 
-* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS pl/PROGRAMMING_LANGUAGE [t/TAG]...` +
-e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 pl/HTML t/friend t/1stYear`
-* *Clear* : `clear`
-* *Delete* : `delete INDEX` +
-e.g. `delete 3`
-* *Edit* : `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [pl/PROGRAMMING_LANGUAGE] [t/TAG]...` +
-e.g. `edit 2 n/James Lee e/jameslee@example.com`
-* *Find* : `find KEYWORD [MORE_KEYWORDS]` +
-e.g. `find James Jake`
-* *List* : `list`
-* *Help* : `help`
-* *Select* : `select INDEX` +
-e.g.`select 2`
-* *History* : `history`
-* *Undo* : `undo`
-* *Redo* : `redo`
-* *Show Dashboard*: `showDB INDEX` +
-e.g. `showDB 3`
-* *Add Milestone* : `addMS i/STUDENT_INDEX d/MILESTONE_DUE_DATE o/DESCRIPTION_OF_MILESTONE` +
-e.g. `addMS i/1 d/23/11/2018 23:59 o/Arrays`
-* *Delete Milestone* : `deleteMS i/STUDENT_INDEX m/MILESTONE_INDEX` +
+[width="60%",cols="20%,<40%,<40%",options="header",]
```

Comment:
Thanks for adding the table! It'll help make our command summary look more readable.